application/features/mfa.feature

```diff
@@ -55,12 +55,21 @@ Feature: MFA
     When I request "/mfa" be created
     Then the response status code should be 200
       And the response should contain a data array with 0 items
-      And an MFA record exists for an employee_id of "123"
+      And an MFA record exists for an employee_id of "123" and has a backup code
       And the following MFA data should be stored:
         | property  | value          |
         | type      | manager        |
         | label     | A Label        |
         | verified  | 1              |
+    When I request "/mfa" be created
+     And I provide the following valid data:
+       | property    | value          |
+       | employee_id | 123            |
+       | type        | manager        |
+       | label       | A Label        |
+    Then the response status code should be 200
+     And the response should contain a data array with 0 items
+     And the same MFA record exists with the same backup code
```

Comment:
A Behat scenario is intended to only have a single "When" (coming after the Given section, and following by the Then section).

Would you mind breaking this "only once" test out into its own scenario?

Reply:
It's also probably only important that the backup code itself is unchanged. I wouldn't compare the MFA records' IDs, since that seems like an implementation detail. If we later change how we implement this in such a way that a new record is created that has the same value, that still accomplishes the goal (and so, I think, shouldn't cause a test to fail).